src/OSPSuite.UI/Views/ParameterIdentifications/ParameterIdentificationFeedbackView.cs

```diff
@@ -69,7 +69,7 @@ public override void InitializeResources()
       {
          base.InitializeResources();
          Caption = Captions.ParameterIdentification.FeedbackView;
-         Icon = ApplicationIcons.ParameterIdentificationVisualFeedback;
+         // Icon = ApplicationIcons.ParameterIdentificationVisualFeedback;
```

Comment:
Icon is missing or why commented out?

Reply:
Should not be commented out. Will revert